.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow runs our benchmarks uploads the results to S3 and sends a notification to
+# slack when it's done.
+name: Benchmark
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs at midnight UTC every day
+  workflow_dispatch:
```

Comment:
Does this make it only run on main?

Reply:
`workflow dispatch` makes it so that there's a button you can click in the actions tab to manually trigger the action. I want it for testing. Could also come in handy if we want to run it for releases as you can choose the branch to run it on.